library/core/src/ffi/c_str.rs

```diff
@@ -169,24 +182,82 @@ impl fmt::Debug for CStr {
     }
 }
 
-#[stable(feature = "cstr_default", since = "1.10.0")]
-impl Default for &CStr {
-    fn default() -> Self {
-        const SLICE: &[c_char] = &[0];
-        // SAFETY: `SLICE` is indeed pointing to a valid nul-terminated string.
-        unsafe { CStr::from_ptr(SLICE.as_ptr()) }
+/// Converts a string or byte literal to a `&'static Cstr`.
```

Comment:
Now this has become outdated :rofl: . 

```suggestion
/// Converts a `const` `'static` string or byte slice to a `&'static Cstr`.
```

Reply:
Well, the macro input argument is now a `literal` though.

Reply:
Good point. It's not neccessary for it to be a literal though, any output from a const fn would work.

Reply:
(But it's 100% fine to have a literal at the start and extend it to general expressions later)

Reply:
Yeah, I'd rather keep it as simple as possible for now.